src/winetricks

```diff
@@ -6682,7 +6685,9 @@ load_dotnet45()
     w_call remove_mono
 
     # See https://appdb.winehq.org/objectManager.php?sClass=version&iId=25478 for Focht's recipe
-    w_call dotnet35
+
+    # Seems unneeded in wine-2.0
```

Comment:
--verify was used, which means the dotnet_verifier was run. That only checks the given .Net version though, not earlier versions.